src/Microsoft.SqlTools.ServiceLayer/LanguageServices/AutoCompleteHelper.cs

```diff
@@ -343,6 +343,14 @@ internal static CompletionItem[] EmptyCompletionList
         }
 
         /// <summary>
+        /// Retrieves the reserved word list 
+        /// </summary>
+        public static string[] GetReservedWordList()
+        {
+            return DefaultCompletionText;
```

Comment:
If you don't want to add a separate assessor, please  add it as a property similar to EmptyCompletionList above.

---
In reply to: [92691614](https://github.com/Microsoft/sqltoolsservice/pull/191#discussion_r92691614) [](ancestors = 92691614)